package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "orval-workspaces",
   "namespace": "@orval",
-  "version": "6.28.2",
```

Comment:
Version updates are performed at the time of release, so there is no need to update each PR.

Reply:
gotcha, I ditched this change